pkg/resource/acl/hooks.go

```diff
@@ -41,6 +41,14 @@ func (rm *resourceManager) validateACLNeedsUpdate(
 	return nil
 }
 
+// aclActive returns true when the status of the given ACL is set to `active`
+func (rm *resourceManager) aclActive(
+	latest *resource,
+) bool {
+	latestStatus := latest.ko.Status.Status
+	return latestStatus != nil && *latestStatus == "active"
```

Comment:
I see there is no constant in the SDK for the `active` status - would you mind moving this to a variable at the top of the file so it's not a magic string.

Reply:
Updated